src/Hamiltonians/abstract.jl

```diff
@@ -232,6 +232,69 @@ function build_sparse_matrix_from_LO(
     end
 end
 
+"""
+    basis = build_basis_only_from_LO(
+        ham, add; cutoff, filter=nothing, sort=false, kwargs...
```

Comment:
```suggestion
        ham, add=starting_address(ham); cutoff, filter=nothing, sort=false, kwargs...
```
Not sure if this needs to be fixed in the other function as well.